conference/src/main/java/com/kyriba/conference/management/domain/exception/LinkedEntityNotFound.java

```diff
@@ -0,0 +1,9 @@
+package com.kyriba.conference.management.domain.exception;
+
+public class LinkedEntityNotFound extends Exception
```

Comment:
The same comments as for another exception.

Reply:
done